cmd/containerize/containerize.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"teleport-exec/cgroups"
+	"teleport-exec/containerize"
+	"time"
+)
+
+func main() {
+	controller := containerize.NewController()
+	defer controller.Close()
+
+	if len(os.Args) < 2 {
+		fmt.Println("Usage: containerize.go command args...")
+		os.Exit(1)
+	}
+
+	// Prepare cgroups support
+	cgroups.Setup()
+	defer cgroups.TearDown()
+
+	// Try to start the command
+	cmd, err := controller.StartCommand(os.Args[1:])
+	if err != nil {
+		log.Fatalln("Error:", err)
+	}
+
+	// Timeout tailing after a while
+	timeout_ctx, timeout_cancel := context.WithTimeout(context.Background(), 15*time.Minute)
+	defer timeout_cancel()
+
+	// Stop the stream when cancelled via Ctrl+C
+	interrupt_ctx, interrupt_cancel := signal.NotifyContext(timeout_ctx, os.Interrupt)
+	defer interrupt_cancel()
+
+	// Start tailing the command output
+	stream, err := cmd.NewLogStream(interrupt_ctx)
+	if err != nil {
+		log.Fatalln("Failed to start a log stream:", err)
+	}
+	defer stream.Close()
+
+	// Stream content until the stream is terminated
+	buffer := make([]byte, 100)
+	fmt.Println(strings.Repeat("-", 80))
+	for {
```

Comment:
Why not use `io.Copy` instead of this loop?

Reply:
It was written before I implemented the Reader interface in the log streaming code and then it just stayed as a loop. Replaced with io.Copy now and it works!